test/Microsoft.IdentityModel.Tokens.Saml.Tests/SamlSecurityTokenHandlerTests.cs

```diff
@@ -315,6 +316,32 @@ public void ValidateToken(SamlTheoryData theoryData)
             ClaimsPrincipal retVal = null;
             try
             {
+                //if (theoryData.TestId.Contains("IssuerSigningKey set"))
```

Comment:
try and use this pattern for creating and using 'context'

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/xml/test/Microsoft.IdentityModel.Tokens.Tests/xml/DelegatingXmlDictionaryReaderTests.cs#L41

Remember to call "TestUtilities.AssertFailIfErrors(context);" outside try catch

Reply:
Fixed